docs/blog-post.md

```diff
@@ -0,0 +1,286 @@
+<h1><b>How to make akka serialization bulletproof</b></h1>
```

Comment:
Pls use upper-case Akka where "akka" is used as a technology name (as opposed to module/library name like `akka-serialization-helper`)

Reply:
Done for all file